docs/threat_model/dataflow.rst

```diff
@@ -0,0 +1,9 @@
+Data Flow Diagram
+=================
+
+The following diagram captures all data flows to and from a SecureDrop deployment.
+
+|SecureDrop data flow diagram|
```

Comment:
If we add the image in this style, it shows based on the width allowed in the theme, and default size is small enough that one may not be able to read the text in the image. If we use the directive below, it becomes a clickable (mouse over on the image gives an option to click) and by just clicking on the image one can open the full size image.

```
.. image:: ../diagrams/SecureDrop_DataFlow.png
   :scale: 100%
```

Reply:
Great catch, @kushaldas! Just tested both locally and @kushaldas's suggestion is indeed more intuitive. +1 for implementing.

Reply:
Agreed, this is way better. I will push the change, excellent suggestion @kushaldas !